test/unit/specs/components/common/TmBalance.spec.js

```diff
@@ -47,7 +47,7 @@ describe(`TmBalance`, () => {
   })
 
   it(`displays unbonded tokens`, () => {
-    expect(wrapper.vm.unbondedAtoms).toBe(`1,230.0000…`)
+    expect(wrapper.vm.unbondedAtoms).toBe(`1,230.000`)
```

Comment:
isn't that misleading as it suggests, that there is no more precision then 3?

Reply:
this was originally from user feedback but i tend to agree with this method. balance.io uses 3 digits. ledger live shows 4. 

the ellipsis are visually very messy / cluttered and are probably worse than just showing all 6 digits.

i think showing 3 digits as display is sufficient for human understanding - but we should show the full 6 digits when transacting (which this PR addresses)